source/common/runtime/runtime_features.cc

```diff
@@ -172,6 +172,9 @@ FALSE_RUNTIME_GUARD(envoy_reloadable_features_dns_nodata_noname_is_success);
 // TODO(abeyad): Evaluate and either remove or make a config knob in
 // https://github.com/envoyproxy/envoy/blob/main/api/envoy/extensions/transport_sockets/tls/v3/tls.proto#L29.
 FALSE_RUNTIME_GUARD(envoy_reloadable_features_quic_disable_client_early_data);
+// TODO(jewertow): Flip to true after implementing support for all major OCI registries,
+// i.e. Azure ACR, AWS ECR, Google Artifact Registry, Docker, Quay.
+FALSE_RUNTIME_GUARD(envoy_reloadable_features_wasm_plugin_enable_fetching_from_oci_regisitry);
```

Comment:
Rather than a runtime guard to decide if oci is enabled, probably a [not-implemented] config makes more sense here.

Reply:
Ok, I reverted this runtime guard.